src/js/config/config.js

```diff
@@ -15,5 +15,10 @@ var config = {
 };
 if (config.environment === "production") {
   config.rootUrl = "dist/";
+} else {
+  try {
+    var configDev = require("./config.dev");
+    config = Object.assign(config, configDev);
+  } catch (e) {}
```

Comment:
This triggers a `no-empty` eslint warning for me. I've tried a couple of different approaches, but I think the try/catch probably is the best solution here - so we should either suppress this warning or add some operation in the catch block.